Management_System/Management_System/Entities/UserBoardEntity.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ManagementSystem.Entities
+{
+    public class UserBoardEntity
+    {
+        public Guid UserId { get; set; }
+        public Guid BoardId { get; set; }
+
+        public UserEntity User { get; set; } = null!;
+        public BoardEntity Board { get; set; } = null!;
+    }
```

Comment:
Is it necessary to have this class?

Reply:
It's necessary for data generation and implementation roles like an ordinary user, manager etc